cmd/nerdctl/run.go

```diff
@@ -196,6 +196,7 @@ func setCreateFlags(cmd *cobra.Command) {
 	cmd.Flags().StringSlice("cap-drop", []string{}, "Drop Linux capabilities")
 	cmd.RegisterFlagCompletionFunc("cap-drop", capShellComplete)
 	cmd.Flags().Bool("privileged", false, "Give extended privileges to this container")
+	cmd.Flags().Bool("privileged-without-host-devices", false, "Don't pass host devices to privileged containers")
```

Comment:
Should this automatically imply `--privileged`?
Or should this fail when `--privileged` is not specified?

Reply:
Updated. IMO it's better to imply `--privileged`, and if both are specified, an error will be returned.